### PR TITLE
Changes to support Podium on GTK

### PR DESCRIPTION
**Pull request by hawkowl**
- WebView.evaluate need to be callback based, since it's asynchronous on GTK
- Fullscreening windows on GTK

### CODE REVIEW

File: src/gtk/toga_gtk/app.py [hunk 1]
```diff
@@ -221,4 +223,7 @@ def open_document(self, fileURL):
         self.interface._documents.append(document)
 
         # Show the document.
-        document.show()
+        self._current_window = document.show()
+
+    def current_window(self):
+        return self._current_window._impl
```
Comment (brutusthebee) [inline src/gtk/toga_gtk/app.py]: At column 1: (W391) blank line at end of file

File: src/gtk/toga_gtk/widgets/webview.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
-from toga_gtk.libs import Gtk, WebKit2
+import time
```
Comment (brutusthebee) [inline src/gtk/toga_gtk/widgets/webview.py]: At column 1: (F401) 'time' imported but unused

File: src/gtk/toga_gtk/widgets/webview.py [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
+
+from toga_gtk.libs import Gtk, WebKit2, Gdk
```
Comment (brutusthebee) [inline src/gtk/toga_gtk/widgets/webview.py]: At column 1: (F401) 'toga_gtk.libs.Gdk' imported but unused

File: src/gtk/toga_gtk/app.py (hunk 1 above)
Comment (brutusthebee) [inline src/gtk/toga_gtk/app.py]: At column 1: (W391) blank line at end of file

File: src/gtk/toga_gtk/widgets/webview.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-from toga_gtk.libs import Gtk, WebKit2
+from toga_gtk.libs import Gtk, WebKit2, Gdk
```
Comment (brutusthebee) [inline src/gtk/toga_gtk/widgets/webview.py]: At column 1: (F401) 'toga_gtk.libs.Gdk' imported but unused

**Comment by hawkowl** [comment]
@freakboy3742 This is the PR I mentioned ^ The tests fail, but that's because I changed the API that I'd take you to look at.

File: src/winforms/toga_winforms/widgets/webview.py [hunk 5]
```diff
@@ -28,8 +28,11 @@ def get_dom(self):
     def set_user_agent(self, value):
         self.native.customUserAgent = value if value else "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10240"
```
Comment (brutusthebee) [inline src/winforms/toga_winforms/widgets/webview.py]: At column 120: (E501) line too long (189 > 119 characters)

File: src/iOS/toga_iOS/widgets/webview.py [hunk 6]
```diff
@@ -50,5 +50,8 @@ def set_user_agent(self, value):
         # self.native.customUserAgent = value if value else "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/603.3.8 (KHTML, like Gecko) Version/10.1.2 Safari/603.3.8"
```
Comment (brutusthebee) [inline src/iOS/toga_iOS/widgets/webview.py]: At column 120: (E501) line too long (179 > 119 characters)

File: src/gtk/toga_gtk/app.py [hunk 7]
```diff
@@ -12,11 +12,17 @@
 from toga.command import GROUP_BREAK, SECTION_BREAK, Command
 from toga.handlers import wrapped_handler
```
Comment (brutusthebee) [inline src/gtk/toga_gtk/app.py]: At column 1: (F401) 'toga.handlers.wrapped_handler' imported but unused

**Pull request merged**